CefSharp.Wpf/DragCursorProvider.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Windows.Input;
+using System.Windows.Interop;
+using CefSharp.Enums;
+using Microsoft.Win32.SafeHandles;
+
+namespace CefSharp.Wpf 
+{
+
+    internal static class DragCursorProvider 
+    {
+        [DllImport("kernel32.dll")]
+        private static extern IntPtr LoadLibrary(string dllToLoad);
+
+        [DllImport("user32.dll")]
+        private static extern IntPtr LoadCursor(IntPtr hInstance, ushort lpCursorName);
+        
+        private static readonly Dictionary<DragOperationsMask, Cursor> DragCursors;
+
+        static DragCursorProvider() 
+        {
+            var library = LoadLibrary("ole32.dll");
+            DragCursors = new Dictionary<DragOperationsMask, Cursor>()
+            {
+                { DragOperationsMask.None, GetCursorFromLib(library, 1) },
```

Comment:
What's the difference between this `None` cursor and `Cursors.None`? Be better to use the built in one where possible

https://docs.microsoft.com/en-us/dotnet/api/system.windows.input.cursors.none?view=netframework-4.7.2

Reply:
Cursors.None is an invisible cursor as the None in drag&drop is a forbidden one.